gcemetadata/lib/gcemetadata/gcemetautils.py

```diff
@@ -26,43 +26,46 @@
 
 from gcemetaExceptions import *
```

Comment:
Agreed, but it is mentioned as part of [pep8](https://www.python.org/dev/peps/pep-0008/#imports) even if `pep8` doesn't raise a warning. The `flake8` wrapper however does raise exception for all instances of wildcard usage + usage of imports that aren't explicitly imported.

E.g.
`F405 'GCEMetadataException' may be undefined, or defined from star imports: gcemetaExceptions`
`F403 'from gcemetaExceptions import *' used; unable to detect undefined names`